common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -408,6 +408,11 @@ public final class LC {
     @Supported
     public static final KnownKey ldap_starttls_required = KnownKey.newKey(true);
 
+    @Reloadable
+    public static final KnownKey clamav_socket_timeout = KnownKey.newKey(2000);
+    @Reloadable
+    public static final KnownKey clamav_scan_data_chunk_size = KnownKey.newKey(2048);
```

Comment:
extensions can define their own LC so why these are added in mailbox and not https://github.com/Zimbra/zm-clam-scanner-store ?
I think we are increasing tight coupling between mailbox and extensions

Reply:
I tried this in another extension, but it does not work and I ended up moving config into zm-mailbox. need to investigate this more.